Add WF Update limits to the kb (#2298)
temporalio/documentation: kb/temporal-platform-limits-sheet.md

    - `limit.numPendingCancelRequests.error`
    - `limit.numPendingChildExecutions.error`
- [Custom Search Attributes limits](/visibility/#custom-search-attributes-limits)
- **Workflow Update limits**

>   - Temporal will limit Workflow Executions to a maximum of 10 concurrently executing Updates. Update requests that would push the system beyond this limit result in an error. This limit can be adjusted globally or on a per-namespace basis via the `history.maxInFlightUpdates` dynamic configuration variable.
>   - Temporal will limit Workflow Executions to a maximum of 2,000 total Updates. Update requests that would push the system beyond this limit result in an error. This limit can be adjusted globally or on a per-namespace basis via the `history.maxTotalUpdates` dynamic configuration variable.

Reviewer: ```suggestion
  - Temporal limits Workflow Executions to a maximum of 10 concurrently executing Updates.
    Update requests that would push the system beyond this limit result in an error.
    You can adjust this limit globally or per Namespace via the `history.maxInFlightUpdates` dynamic configuration variable.
  - Temporal limits Workflow Executions to a maximum of 2,000 total Updates.
    Update requests that would push the system beyond this limit result in an error.
    You can adjust this limit globally or per Namespace via the `history.maxTotalUpdates` dynamic configuration variable.
```
